### PR TITLE
Pypi format fix

### PR DESCRIPTION
**Pull request by etiennerichart**
View at https://test.pypi.org/project/curtsies/0.3.2/

### CODE REVIEW

**Comment by thomasballinger** [comment]
Oh much better, thanks! I wonder what all that other junk in that method is for, I think you should delete it all. Maybe PyPI didn't used to work with md files or something.

File: setup.py [hunk 1]
```diff
@@ -12,31 +12,11 @@ def version():
...
-        long_description = long_description.replace("\r", "")
-    return long_description
-
-
 setup(
     name="curtsies",
     version=version(),
     description="Curses-like terminal wrapper, with colored strings!",
-    long_description=get_long_description(),
+    long_description=open("readme.md").read(),
```
Comment (thomasballinger) [inline setup.py:19]: To make this work on Windows we should keep the `encoding='utf-8'` argument in the `open()` call.

**Comment by thomasballinger** [comment]
Whoops, turns out my change doesn't work with Python 2.

**Comment by etiennerichart** [comment]
It seems that it is also not working with pypy and pypy3. Also, after deleting the method get_long_description I believe we no longer need the import io.

**Pull request merged**